eater_member_card/report/member_card_template.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+
+    <record id="member_card.report_member_card" model="ir.actions.report">
+        <field name="report_name">eater_member_card.member_card_template</field>
+    </record>
+
+
+
+
+    <template id="member_card_template">
+        <!-- replace the member_card template to add worker information.
+            Cannot use template inheritance because the worker information is in partner variable,
+        which is set in the foreach loop -->
```

Comment:
I'm surprised by this assertion, it works in other report inheritance 🤔

Reply:
I could not make it work, do you have an example ?

Reply:
@robinkeunen up :)